microservices-orders/start-ccloud.sh

```diff
@@ -18,9 +18,9 @@ printf "\n====== Create new Confluent Cloud stack\n"
 export EXAMPLE="microservices-orders"
 ccloud::create_ccloud_stack true
 
-SERVICE_ACCOUNT_ID=$(confluent kafka cluster list -o json | jq -r '.[0].name' | awk -F'-' '{print $4 "-" $5;}')
+SERVICE_ACCOUNT_ID=$(ccloud:get_service_account_from_current_cluster_name)
 if [[ "$SERVICE_ACCOUNT_ID" == "" ]]; then
-  printf "\nERROR: Could not determine SERVICE_ACCOUNT_ID from 'confluent kafka cluster list'. Please troubleshoot, destroy stack, and try again to create the stack.\n"
+  printf "\nERROR: Could not determine SERVICE_ACCOUNT_ID from 'confluent kafka cluster describe'. Please troubleshoot, destroy stack, and try again to create the stack.\n"
```

Comment:
Minor - the error message being constructed here knows that `ccloud:get_service_account_from_current_cluster_name` is executing `confluent kafka cluster describe` internally. If that function is ever changed, this error message would also need to be updated.

Reply:
Agree, as evidenced by the change here.  IMO I'm ok with leaving it as-is for now, as it doesn't do worse than it is today, but agree it's not great practice.